docs/pages/docs/custom-schemas.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Custom Schemas
+description: Learn how to create custom schemas for your BlockNote editor
+---
+
+# Custom Schemas (advanced)
+
+By default, BlockNote documents support different kind of blocks (paragraphs, headings, lists, etc.), inline content (text, links) and text styles (bold, italic, underline, etc.). See xxxx.
```

Comment:
Not sure we need to put examples of blocks, inline content, and styles here since we do it again in each subsection